### ISSUE TITLE
A11y: Make the tooltip of Toolbar buttons and Icon buttons dismissable

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#54273]
**Steps to reproduce**:
In `Explore Page`:
1. Go here: https://play.grafana.org/explore
2. Hover over the "Split" button of the Toolbar. Hover over the icons buttons of the QueryOperationRowHeader ("Duplicate query", "Disable/enable query", "Remove query" and "Drag and drop to reorder").
3. After clicking on the "Split" button, hover on the "Close" one.

In panels:
1. Go here: https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&editPanel=4
2. Hover over "Settings", "Discard", "Save", "Apply" or "Data Source Help" icon button under Query

This issue is related to `Toolbar buttons` and `Icon buttons`.
This issue was triggered by #53844 and #53888.

**Actual Result**:
The tooltip is displayed until the user moves pointer hover or keyboard focus.

**Expected Result**
The user should have a mechanism available to dismiss the tooltip to fulfil the criteria ( dismissible, hoverable and persistent).

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
https://www.w3.org/WAI/WCAG22/Understanding/content-on-hover-or-focus

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:
